sympy/core/tests/test_args.py

```diff
@@ -498,7 +497,7 @@ def test_sympy__sets__fancysets__Reals():
 
 def test_sympy__sets__fancysets__ImageSet():
     from sympy.sets.fancysets import ImageSet
-    from sympy import S, Lambda, Symbol
+    from sympy import S, Symbol
```

Comment:
I needed to import`` Lambda``, which was imported multiple times already. So, I took it out. Well, I will try to cleanup other imports as well. :+1: